Guide/form.markdown

```diff
@@ -5,8 +5,346 @@
 
 ## Introduction
 
+In IHP Forms are an essential way to interact with your application. Dealing with a lot of form markup can quickly become complex because of the need to deal with consistent styling and especially when dealing with lots of validation. IHP provides helpers to generate form markup to help you deal with the complexity.
+
+By default forms in IHP follows the class names used by Bootstrap 4. Therefore the forms work with Bootstrap 4 out of the box. Of course the default form generation can be customized to support other CSS frameworks.
```

Comment:
How/where can it be customized for different CSS frameworks?

Reply:
Added a section for this. This feature basically is still in todo. While technically it's already possible, we still have to allow the user to change the rendering in a global way.